pr-data.csv

```diff
@@ -2819,7 +2820,7 @@ https://github.com/Huawei/vCenter_Plugin_DME,fa077ef3c8ee052afc01084bc5aaae7c3b4
 https://github.com/HubSpot/hbase,102c2bb129dea33d2de928b4888e3e09fd0e04ca,hbase-server,org.apache.hadoop.hbase.io.hfile.TestLruAdaptiveBlockCache.testBackgroundEvictionThread,ID,,,
 https://github.com/HubSpot/hbase,102c2bb129dea33d2de928b4888e3e09fd0e04ca,hbase-server,org.apache.hadoop.hbase.master.TestRegionsRecoveryChore.testRegionReopensWithLessThreshold,ID,,,
 https://github.com/HubSpot/hbase,102c2bb129dea33d2de928b4888e3e09fd0e04ca,hbase-server,org.apache.hadoop.hbase.master.TestRegionsRecoveryChore.testRegionReopensWithStoreRefConfig,ID,,,
-https://github.com/HubSpot/hbase,102c2bb129dea33d2de928b4888e3e09fd0e04ca,hbase-server,org.apache.hadoop.hbase.monitoring.TestTaskMonitor.testClone,ID,Opened,https://github.com/apache/hbase/pull/4876,
+https://github.com/HubSpot/hbase,102c2bb129dea33d2de928b4888e3e09fd0e04ca,hbase-server,org.apache.hadoop.hbase.monitoring.TestTaskMonitor.testClone,ID,,,
```

Comment:
It' better to add some note here indicating that you've fixed the flaky test in the original repo